src/CommunityVoices/App/Website/Bootstrap/FrontController.php

```diff
@@ -46,18 +46,30 @@ public function __construct($router, $dispatcher, $injector, $logger)
         $this->dispatcher = $dispatcher;
         $this->injector = $injector;
         $this->logger = $logger;
+
     }
 
     public function doRequest($request)
     {
         $this->loadProviders($request);
 
-        // try {
+        // if the server is local, we want to be able to see the error stack,
+        // but on website we want to have an error page instead
+        if ($this->router->isLocal()) {
+
+          $this->router->route($request);
+          $this->dispatcher->dispatch($request)->send();
+        }
+
```

Comment:
A blank line between an `if` and `else` is not good for my brain.  Sorry, this is a really minor thing.

Reply:
Resolving because this line no longer exists.